universal/include/userver/utils/strong_typedef.hpp

```diff
@@ -365,7 +365,7 @@ std::string ToString(const StrongTypedef<Tag, std::string, Ops>& object) {
 }
 
 template <typename Tag, typename T, StrongTypedefOps Ops,
-          typename = std::enable_if_t<meta::kIsInteger<T>>>
+          typename = std::enable_if_t<meta::kIsNumber<T>>>
```

Comment:
std::to_string for floating point types would change it's behavior in C++26 to not use locales and to produce more precise results.

Add a new function and use fmt::format in it, just like C++26 does
```
template <typename Tag, typename T, StrongTypedefOps Ops
                   typename = std::enable_if_t<std::is_flotaing_point_v<T>>>
std::string ToString(const StrongTypedef<Tag, T, Ops>& object) {
  impl::strong_typedef::CheckIfAllowsLogging<
      StrongTypedef<Tag, std::string, Ops>>();
  return fmt::format("{} ", object.GetUnderlying());
}
```

Reply:
Done